CONTRIBUTING.md

```diff
@@ -304,17 +304,30 @@ Check in whichever of the files generated by `pkgdown` seem relevant to you. Any
 
 ### Test on latest development version of R (i.e. "R-Devel") 
 
-This is a CRAN requirement.  [This docker based process](https://alexandereckert.com/post/testing-r-packages-with-latest-r-devel/) from Alexander Eckert is very useful.
-
-### Submit to CRAN
-
-Build the package tarball by running the following
+This is a CRAN requirement, and yes, it's a pain.  Luckily, Dirk Eddelbuettel and friends maintain docker images to make this process (a little) easier.  Specifically, the [rocker/r-devel](https://hub.docker.com/r/rocker/r-devel) image provides a great jumping off point.  Note, this image is rebuilt every few days, so if you _absolutely need_ to have _bleeding edge_ r-devel and have a extra hour to spare, follow [this docker process](https://alexandereckert.com/post/testing-r-packages-with-latest-r-devel/) from Alexander Eckert. 
 
+1. Navigate to the base directory of the package. 
+2. Build the r-devel pkgnet docker image and run CRAN checks via these commands: 
 ```
-R CMD BUILD .
+# On Windows
+docker build -f release_testing\dockerfile -t pkgnet_rdevel . 
+docker run --rm -ti -v %cd%:/RPackage pkgnet_rdevel ./r_devel_check.sh
+
+# On Mac
+docker build -f release_testing\dockerfile -t pkgnet_rdevel . 
+docker run --rm -ti -v `pwd`:/RPackage pkgnet_rdevel ./r_devel_check.sh
```

Comment:
I think you should make it `$(pwd)` instead of backticks. backticks can get misinterpreted in some settings

Reply:
Thanks.  I'm actually doing this release on a windows laptop, so didn't confirm that command.